apps/Gruntfile.js

```diff
@@ -573,6 +684,14 @@ module.exports = function (grunt) {
     'karma:integration'
   ]);
 
+  grunt.registerTask('codeStudioTest', [
+    'newer:messages',
+    'exec:convertScssVars',
+    'newer:copy:static',
+    'concat',
+    'karma:codeStudio'
+  ]);
```

Comment:
No significant reason. The code is still fairly separate at this time, and it kind of made sense to me for the tests to be too.